spec/models/manageiq/providers/amazon/regions_spec.rb

```diff
@@ -15,4 +15,19 @@
       expect(described_class.names).not_to include('us-gov-west-1')
     end
   end
+
+  context "adds additional regions from settings" do
+    it "includes regions from additional_regions" do
+      custom_region = {
+        "my-custom-region" => {
```

Comment:
e.g. Putting in the config as a string with

```yaml
:ems
  :ems_amazon:
    :additional_regions:
    - my-custom-region:
      :name: My Custom Region
      :hostname: localhost
      :description: Custom Region
```

Then pulling it back out you can see it's a Symbol:

```
::Settings.ems.ems_amazon.additional_regions[0].to_h
# => {:"my-custom-region"=>nil,
#     :name=>"My Custom Region",
#     :hostname=>"localhost",
#     :description=>"Custom Region"}
```